src/tests/blueprint/t_blueprint_mesh_utils.cpp

```diff
@@ -164,18 +164,18 @@ TEST(conduit_blueprint_mesh_utils, shapetype)
 //-----------------------------------------------------------------------------
 TEST(conduit_blueprint_mesh_utils, adjset_validate_element_0d)
 {
-    conduit::Node root, info;
+    conduit::Node root, opts, info;
     create_2_domain_0d_mesh(root, 0, 1);
     save_mesh(root, "adjset_validate_element_0d");
-    bool res = conduit::blueprint::mesh::utils::adjset::validate(root, "main_adjset", info);
+    bool res = conduit::blueprint::mesh::utils::adjset::validate(root, "main_adjset", opts, info);
```

Comment:
I asked my earlier question about if we needed to have opts be a mandatory part of the API because it seems like for all of these tests we now have to add opts in when they are unused.

Reply:
I removed opts from all places where a default value will suffice.